tests/bsim/bluetooth/audio/src/bap_broadcast_sink_test.c

```diff
@@ -487,6 +561,39 @@ static void test_broadcast_delete_inval(void)
 	}
 }
 
+static void test_start_adv(void)
+{
+	const struct bt_data ad[] = {
+		BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+		BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_BASS_VAL),
+			      BT_UUID_16_ENCODE(BT_UUID_PACS_VAL)),
+		BT_DATA_BYTES(BT_DATA_SVC_DATA16, BT_UUID_16_ENCODE(BT_UUID_BASS_VAL)),
```

Comment:
Is this one correct? It looks like Service Data declaration with no actual data?